{"uuid": "3f587b93-c519-47eb-a1ea-32647389fc31", "children": ["965f0abf-f99a-4596-bc6e-f8fb05f386c8"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_INTERNET_DISCONNECTED\n  (Session info: chrome=128.0.6613.86)\nStacktrace:\n\tGetHandleVerifier [0x00007FF60FA1B632+29090]\n\t(No symbol) [0x00007FF60F98E6E9]\n\t(No symbol) [0x00007FF60F84B1CA]\n\t(No symbol) [0x00007FF60F843204]\n\t(No symbol) [0x00007FF60F834179]\n\t(No symbol) [0x00007FF60F835F42]\n\t(No symbol) [0x00007FF60F83443F]\n\t(No symbol) [0x00007FF60F833CD1]\n\t(No symbol) [0x00007FF60F833C10]\n\t(No symbol) [0x00007FF60F831AD3]\n\t(No symbol) [0x00007FF60F83214C]\n\t(No symbol) [0x00007FF60F84E231]\n\t(No symbol) [0x00007FF60F8E73FE]\n\t(No symbol) [0x00007FF60F8C66EA]\n\t(No symbol) [0x00007FF60F8E65D9]\n\t(No symbol) [0x00007FF60F8C6493]\n\t(No symbol) [0x00007FF60F8909B1]\n\t(No symbol) [0x00007FF60F891B11]\n\tGetHandleVerifier [0x00007FF60FD3883D+3294125]\n\tGetHandleVerifier [0x00007FF60FD84423+3604371]\n\tGetHandleVerifier [0x00007FF60FD7A2E7+3563095]\n\tGetHandleVerifier [0x00007FF60FAD6F16+797318]\n\t(No symbol) [0x00007FF60F99986F]\n\t(No symbol) [0x00007FF60F995454]\n\t(No symbol) [0x00007FF60F9955E0]\n\t(No symbol) [0x00007FF60F984A7F]\n\tBaseThreadInitThunk [0x00007FFB1FBC257D+29]\n\tRtlUserThreadStart [0x00007FFB1FF8AF28+40]\n\n", "trace": "  File \"C:\\Users\\Nithya\\Desktop\\python_pom_framework\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\Nithya\\Desktop\\python_pom_framework\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Nithya\\Desktop\\python_pom_framework\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Nithya\\Desktop\\python_pom_framework\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1139, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Nithya\\Desktop\\python_pom_framework\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 890, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Nithya\\Desktop\\python_pom_framework\\pytest_demowebshop\\Tests\\conftest.py\", line 8, in driver\n    driver.get(\"https://demowebshop.tricentis.com/\")\n  File \"C:\\Users\\Nithya\\Desktop\\python_pom_framework\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 363, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"C:\\Users\\Nithya\\Desktop\\python_pom_framework\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Nithya\\Desktop\\python_pom_framework\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1724913744940, "stop": 1724913747684}], "start": 1724913744940, "stop": 1724913747950}